### ISSUE TITLE
Is AttributionCallback is called when the app is updated?

### ISSUE THREAD

**Issue by hiroshi-imamasu** [adjust/android_sdk#380]
I'm using AttributionCallback on myApp. 

In Docs, they say that "Current attribution information is only available after our backend tracks the app install". 

Does it means the callback is not called when the app is updated ?? 
And It is also true in IOS SDK??

Thanks.

**Comment by uerceg** [comment]
Hi @hiroshi-imamasu 

App update is not an action which our SDK and backend "care" in particular. Attribution callback will be triggered when ever SDK gets informed about attribution change for the user. SDK keeps attribution information locally stored inside of your app and keeps track of it. If session which maybe follows your app opening after user has updated the app leads to change in user's attribution, then in that case we will trigger attribution callback because we got new information about the attribution. But this might as well happen on any other session and it has nothing to do with the fact that your user updated an app.

So TLDR - updating an app and opening it after an update does not trigger attribution callback (unless attribution changed for the user in exactly that moment which is again unrelated to app update action).

Every single SDK on every platform behaves in this way.

Cheers

**Comment by hiroshi-imamasu** [comment]
@uerceg Thank you!!! You make my day!!
